content/blog/2025-03-27-valkey-8-1-0-rc1.md

```diff
@@ -0,0 +1,282 @@
+
+Similarly, the BITCOUNT operation has been improved up to [514%](https://github.com/valkey-io/valkey/pull/1741) using AVX2 on x86.
+
+### Other Performance Enhancements
+
+Active Defrag has been improved to eliminate large latencies to below [1ms](https://github.com/valkey-io/valkey/pull/1242) and anti-starvation protection has been introduced in the presence of long-running commands.
+
+Periodic processing/monitoring of clients has been moved off of the 100ms `serverCron()` interval loop, reducing system [overhead](https://github.com/valkey-io/valkey/pull/1387)
+ when large numbers of clients are present. 
+
+## Observability
+
+There are also several improvements to the observability of the system behavior in Valkey 8.1.
+
+### Log Improvements
+
+Valkey 8.1 brings new options to the format of the log file entries as well as the way timestamps are recorded in the log file. This makes it easier to consume the log files by log collecting systems. 
+
+The format of the log file entries is controlled by the `log-format` parameter, where the default is the existing format :
+
+- `legacy`: the default, traditional log format
+- `logfmt`: a structured log format; see https://www.brandur.org/logfmt
+
+The formatting of the timestamp of the log file entries is controlled by the `log-timestamp-format` parameter, where the default is the existing format: 
+
+- `legacy`: default format
+- `iso8601`: ISO 8601 extended date and time with time zone, of the form yyyy-mm-ddThh:mm:ss.sss±hh:mm
+- `milliseconds`: milliseconds since the epoch
+
+An example of the format `logfmt` with the timestamp format `iso8601`:
+
+```
+pid=3883950 role=primary timestamp="2025-03-26T05:17:30.402-04:00" level=notice message="oO0OoO0OoO0Oo Valkey is starting oO0OoO0OoO0Oo
+```
+
+An example of the `legacy` format which remains the same:
+
+```
+3223494:M 18 Mar 2025 14:47:42.070 * oO0OoO0OoO0Oo Valkey is starting oO0OoO0OoO0Oo:
+```
+
+*Note*: as can be observed from the examples, using both the `logfmt` and `iso8601` format uses around 60% more space, so disk space should be considered when implementing these.
+
+### Extending the Slowlog to Commandlog
+
+Valkey has long had the capability to record slow commands during execution time based on the threshold set with  `slowlog-log-slower-than` parameter to keep the last `slowlog-max-len` entries.
+
+The new `COMMANDLOG` feature expands the recording of slow commands beyond execution time by adding recording of large requests and replies. These metrics do not necessarily affect execution time but do affect the overall round trip to the application and impact network usage.
+
+The following parameters configure the recording of `COMMANDLOG`:
```

Comment:
similar to my comments on the log format, I think the key message of  a release blog like this is to excite the audience. The "instructions" or "manual" are important too but can dilute the key message.